hifireg/registration/templates/registration/index.html

```diff
@@ -2,15 +2,16 @@
 
 {% block card_content %}
 
-<div class="field">
-    <p>Hi, {{ view.request.user.first_name }}!</p>
-    <p>Thanks for logging in.</p>
-</div>
+<p>Hi, {{ view.request.user.first_name }}!</p>
+<p>We look forward to dancing with you at this year's HiFi Fusion!</p>
+{% include "utils/form_fields.html" %}
 
 <div class="buttons">
-    {% include "utils/button.html" with button=view.register_button class="is-primary" %}
-    {% include "utils/button.html" with button=view.account_button %}
-    {% include "utils/button.html" with button=view.invoices_button %}
+    {% include "utils/button.html" with button=view.register_button %}
```

Comment:
Put this class back.
```suggestion
    {% include "utils/button.html" with button=view.register_button class="is-primary" %}
```

Reply:
Done

Reply:
done